### PR TITLE
AWS AccessKey and SecretKey set to 129 char long

### LINKED ISSUE #4582: Allow longer S3 Access Key and Secret Key
### My actions before raising this issue
- [X] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [X] Searched [past issues](/issues)

### Expected Behaviour

I would like to connect to an AWS S3 compatble service (Cloud Storage) served from an on-premise storage.

### Current Behaviour

Both AccessKey and SecretKey fields are truncated to match AWS's number of characters. 

Our AWS S3 compatible service uses 129 character long AccessKey and SecretKey. 

### Possible Solution

Modify Acces Key and Secret Key fields to allow entering 129 characters long keys.

### Steps to Reproduce (for bugs)

1. Click on Cloud Storage
1. Click on Add (+)
1. Enter Bucket Name, Endpoint, Access Key and Secret Key (both are truncated)

### Your Environment

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05): Version:  20.10.12
- Are you using Docker Swarm or Kubernetes? No
- Operating System and version (e.g. Linux, Windows, MacOS):  macOS 12.0.1
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    [Wed Apr 20 22:39:10.737344 2022] [wsgi:error] [pid 555:tid 275483744000] [remote 172.20.0.4:35558] ERROR - 2022-04-20 22:39:10,735 - serializers - Cannot create resource datacenter with specified credentials. Access forbidden.
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

### PR DESCRIPTION
**Pull request by sebagarayco**
### Checklist

- [X] I submit my changes into the `develop` branch
~~- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file~~
~~- [] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
- [X] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))  ----> Closes #4582 
~~- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [X] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
~~- [ ] I have updated the license header for each file (see an example below)~~

```python
# Copyright (C) 2022 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by nmanovic** [comment]
@sebagarayco , thanks for the contribution! Please expect delays in our review for now.

File: cvat-ui/src/components/create-cloud-storage-page/cloud-storage-form.tsx [hunk 1]
```diff
@@ -74,8 +74,8 @@ export default function CreateCloudStorageForm(props: Props): JSX.Element {
     const fakeCredentialsData = {
         accountName: 'X'.repeat(24),
         sessionToken: 'X'.repeat(300),
-        key: 'X'.repeat(20),
-        secretKey: 'X'.repeat(40),
+        key: 'X'.repeat(129),
```
Comment (bsekachev) [inline cvat-ui/src/components/create-cloud-storage-page/cloud-storage-form.tsx:77]: Could you explain this change, please?

**Comment by Marishka17** [comment]
Hi, @sebagarayco, thanks for the contribution! Are you still going to contribute to us? If so, could you please resolve conflicts in this PR?

**Comment by bsekachev** [comment]
I will close the PR for now. Please, do not hesitate to reopen if you are ready to respond in the PR.

**Pull request closed without merging**